packages/account-v2/src/hooks/useFormattedLoginHistoryData.ts

```diff
@@ -0,0 +1,49 @@
+import dayjs from 'dayjs';
+import UAParser from 'ua-parser-js';
+
+type TData = {
+    action: string;
+    browser: string;
+    date: string;
+    id: number;
+    ip: string;
+    status: string;
+}[];
+
+export const useFormattedLoginHistoryData = loginhistorydata => {
```

Comment:
please fix the type issue, Build is getting failed because of this

Reply:
```suggestion
import { LoginHistory } from '@deriv/api-types';
...

export const useFormattedLoginHistoryData = (loginHistory: LoginHistory) => {
```